### DISCUSSION TITLE
[A-L] (2023/24) Foglio 1 - Esercizio 2

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/calcolo-delle-probabilita#133] [Esercizi - Bertini]
![Es 2](https://github.com/sapienzastudentsnetwork/calcprob2324/assets/107944476/d1a75847-9295-4ddd-aca9-6140621fcbbe)

**Comment by Elia-Belli** [comment]
Per dimostrazione l'equivalenza delle 3 affernazioni possiamo dimostrare che la **i** implica la **ii**, che la **ii** implica la **iii** e che la **iii** implica la **i**:
- Dimostriamo $A \subset B \Rightarrow A \cap B = A$:
per definizione di intersezione $A \cap B = \\{x | x \in A \land x \in B\\}$
siccome per ipotesi $\forall a \in A \Rightarrow a \in B$ ogni elemento di $A$ è anche in $B$, quindi $A \cap B = A$
- $A \cap B = A \Rightarrow A \cup B = B$
per ipotesi $\forall a \in A \Rightarrow a \in (A \cap B)$, per definizione di intersezione $\forall a \in A \Rightarrow a \in B$, allora siccome $B$ contiene già tutti gli elementi di $A$ l'unione non aggiunge nuovi elementi, si dimostra quindi $A \cup B = B$
- $A \cup B = B \Rightarrow A \subset B$
per ipotesi l'unione $A \cup B$ non aggiunge nuovi elementi a $B$, significa quindi che $\forall a \in A \Rightarrow a \in B$, ovvero la definizione di sottoinsieme, si dimostra quindi $A \subset B$

**Comment by CiottoloMaggico** [comment]
Analogo @Elia-Belli
